search/search-client-query-transformer/src/main/java/io/camunda/search/clients/SearchClients.java

```diff
@@ -98,7 +99,8 @@ public List<AuthorizationEntity> findAllAuthorizations(final AuthorizationQuery
             transformers,
             new DocumentAuthorizationQueryStrategy(this),
             securityContext);
-    return executor.findAll(filter, AuthorizationEntity.class);
+    return executor.findAll(
+        filter, io.camunda.webapps.schema.entities.usermanagement.AuthorizationEntity.class);
```

Comment:
It would be great if we could find a naming convention that didn't require qualification. That's not only going to help when reading and navigating the code, but also when we talk about it. A few options (for the webapps.schema entities):
* `FooExporterEntity`
* `ExportedFoo`
* `FooIndexEntity`

Reply:
Yes, I agree with the proposal
I preferred not to do it in these pull requests to reduce the number of changed files, but we can do that afterward